doc/README_Source.md

```diff
@@ -2,144 +2,168 @@
 
-### Build the connector
-Download the repository and build it
-```shell script
-mvn clean package
-```
+Alternatively, in the connectors page, you can upload the JSON file from earlier by using the `Upload connector config file` option.
 
-### Using docker
-Create variables.env file
-```properties
-CONNECT_BOOTSTRAP_SERVERS={YOUR.EVENTHUBS.FQDN}:9093 # e.g. namespace.servicebus.windows.net:9093
-CONNECT_REST_PORT=8083
-CONNECT_GROUP_ID=connect-cluster-group
+![Upload connector config](./images/upload-connector-config.png "Upload connector config")
 
-CONNECT_CONFIG_STORAGE_TOPIC=connect-cluster-configs
-CONNECT_OFFSET_STORAGE_TOPIC=connect-cluster-offsets
-CONNECT_STATUS_STORAGE_TOPIC=connect-cluster-status
+#### Create connector using REST API
 
-CONNECT_CONFIG_STORAGE_REPLICATION_FACTOR=1
-CONNECT_OFFSET_STORAGE_REPLICATION_FACTOR=1
-CONNECT_STATUS_STORAGE_REPLICATION_FACTOR=1
+Create the source connector using the Connect REST API
 
-CONNECT_KEY_CONVERTER=org.apache.kafka.connect.json.JsonConverter
-CONNECT_VALUE_CONVERTER=org.apache.kafka.connect.json.JsonConverter
-CONNECT_VALUE_CONVERTER_SCHEMAS_ENABLE=false
+```bash
+# Curl to Kafka connect service
+curl -H "Content-Type: application/json" -X POST -d @<path-to-JSON-config-file> http://localhost:8083/connectors
+```
 
-CONNECT_INTERNAL_KEY_CONVERTER=org.apache.kafka.connect.json.JsonConverter
-CONNECT_INTERNAL_VALUE_CONVERTER=org.apache.kafka.connect.json.JsonConverter
+### Insert document in to Cosmos DB
 
-CONNECT_REST_ADVERTISED_HOST_NAME=connect
+Use [Cosmos DB setup guide](./CosmosDB_Setup.md) to create and set up Cosmos DB Instance.
+* Sign into the [Azure portal](https://portal.azure.com/learn.docs.microsoft.com) using the account you activated.
+* On the Azure portal menu (left hand side blue lines at the top), select All services.
+* Select Databases > Azure Cosmos DB. Then select the DB you just created, click Data Explorer at the top.
+* To create a new JSON document, in the SQL API pane, expand `kafka`, select Items, then select New Item in the toolbar.
+* Now, add a document to the container with the following structure. Paste the following sample JSON block into the Items tab, overwriting the current content:
+  ``` json
+  {
+   "id": "2",
+   "productId": "33218897",
+   "category": "Women's Outerwear",
+   "manufacturer": "Contoso",
+   "description": "Black wool pea-coat",
+   "price": "49.99",
+   "shipping": {
+       "weight": 2,
+       "dimensions": {
+       "width": 8,
+       "height": 11,
+       "depth": 3
+      }
+    }
+  }
+  ```
+* Select Save.
+* Confirm the document has been saved by clicking Items on the left-hand menu.
+
+### Confirm data written to Kafka Topic
+
+* Open Kafka Topic UI on http://localhost:9000
+* Select the Kafka topic `apparels` you created
+* Validate document that was inserted in to Cosmos DB in previous step.
```

Comment:
"Verify that the document inserted in to Cosmos DB earlier appears in the Kafka topic. "